### DISCUSSION TITLE
Expand/contract editing window when you open sidebars

### DISCUSSION THREAD

**Discussion by lolbat** [notaapp/talk#378] [Feature Requests] [answered]
I like to have a compact window when I am writing but when I open the sidebars this results in a very cramped UI.

![CleanShot 2023-03-02 at 10 43 15@2x](https://user-images.githubusercontent.com/4564571/222509437-c6cdbea4-26e5-48ef-984c-56cd959aa0da.jpg)

Would it be possible for Nota to remember the editing widow size and then resize the entire window (up and down) when you open and close the sidebars?

Currently I need to resize it manually and then again when I close the sidebars

**Comment by astoilkov** [comment, marked as answer]
I think a similar issue exists — [Window resize when showing sidebars](https://github.com/notaapp/talk/issues/184). Did I understand correctly that these are similar? Do you think there are differences?

**Comment by lolbat** [comment]
It is the same issue.
